fetch.bs

```diff
@@ -2689,9 +2743,25 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
      <a for=request>response tainting</a> to
      "<code>opaque</code>".
 
-     <li><p>Return the result of performing a <a>scheme fetch</a>
-     using <var>request</var>.
+     <li><p>Let <var>noCorsResponse</var> be the result of performing a <a>scheme fetch</a> using
+     <var>request</var>.
      <!-- file URLs end up here as they are not same-origin typically. -->
+
+     <li><p>If the <a>CORB check</a> with <var>request</var> and <var>noCorsResponse</var> returns
+     <b>allowed</b>, then return <var>noCorsResponse</var>.
+
+     <li><p>If <var>noCorsResponse</var> is a <a>filtered response</a>, then return
+     <var>noCorsResponse</var>.
```

Comment:
This needs to move one step up I think (and could therefore be combined again as I did it).